src/services/uniswap.js

```diff
@@ -0,0 +1,72 @@
+import Ethereum from '../services/eth';
+const uni = require('@uniswap/sdk')
+const ethers = require('ethers')
+const proxyArtifact = require('../static/uniswap_v2_router_abi.json')
+const debug = require('debug')('router')
+
+const GAS_LIMIT = 1200000  //120000000
+
+export default class Uniswap {
+  constructor (network = 'kovan') {
+    // network defaults to KOVAN
+    const providerUrl = process.env.ETHEREUM_RPC_URL
+    this.network = process.env.UNISWAP_NETWORK
+    this.chainID = uni.ChainId.KOVAN
+    this.provider = new ethers.providers.JsonRpcProvider(providerUrl)
+    this.exchangeProxy = process.env.UNISWAP_PROXY
```

Comment:
For Uniswap, this variable should be called `router` and not `exchangeProxy`, since it's the address of the contract that we're sending the asset. Uniswap currently uses the `0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D ` address listed on [this page](https://uniswap.org/docs/v2/smart-contracts/router02) for both mainnet and testnets.

Reply:
Done!